File: core/src/main/java/com/bing/lan/core/business/service/VerifyCode.java
package com.bing.lan.core.business.service;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@ToString
public class VerifyCode {
    public String phoneNumber;
    public String randomCode;
    public Date lastSendTime;
    public String content;

    public VerifyCode() {
        super();
    }

    public VerifyCode(String phoneNumber, String randomCode, Date lastSendTime,
            String content) {
        super();
        this.phoneNumber = phoneNumber;
        this.randomCode = randomCode;
        this.lastSendTime = lastSendTime;
        this.content = content;
    }
}

File: core/src/main/kotlin/com/bing/lan/core/business/service/VerifyCode.kt
package com.bing.lan.core.business.service

import lombok.Getter
import lombok.Setter
import lombok.ToString
import java.util.*

@Setter
@Getter
@ToString
class VerifyCode {
    lateinit var phoneNumber: String
    lateinit var randomCode: String
    lateinit var lastSendTime: Date
    lateinit var content: String

    constructor() : super() {}

    constructor(phoneNumber: String, randomCode: String, lastSendTime: Date,
                content: String) : super() {
        this.phoneNumber = phoneNumber
        this.randomCode = randomCode
        this.lastSendTime = lastSendTime
        this.content = content
    }
}
